diskutil.py

```diff
@@ -280,6 +280,9 @@ def getDiskDeviceSize(dev):
         return int(__readOneLineFile__("/sys/block/%s/device/block/size" % dev))
     elif os.path.exists("/sys/block/%s/size" % dev):
         return int(__readOneLineFile__("/sys/block/%s/size" % dev))
+    else:
+        assert 0, "%s not found as %s or %s" % (dev, "/sys/block/%s/device/block/size",
```

Comment:
Why `assert 0` rather than raising an exception directly?

Reply:
A cursory glance was showing several `assert 0` already, but they are a different kind indeed, and even a buggy use of assert.
Good point.